### ISSUE TITLE
Send image and text in same message?

### ISSUE THREAD

**Issue by luciano22** [venomous0x/WhatsAPI#1112]
It's possible with WhatsApi send an image and text in same message like in whatsapp plataform?

**Comment by mgp25** [comment]
@luciano22 yes its possible. Use WhatsAPI Official code. https://github.com/mgp25/WhatsAPI-Official

All its explained in the wiki

Regards

**Comment by hablas** [comment]
you can send the text as a caption
search for " caption " in the wiki
